src/net/constants.rs

```diff
@@ -17,3 +18,5 @@ pub const DEFAULT_MTU: u16 = 1452;
 /// - Generating crc32 for the packet header.
 /// - Validating if arriving packets have the same protocol version.
 pub const PROTOCOL_VERSION: &str = "laminar-0.1.0";
+/// The size of the pool wherein packets will be stored on receive.
+pub const PACKET_POOL_SIZE: usize = 1000;
```

Comment:
Should probably be configurable

Reply:
Actually can remove it, don't need it anymore. I'll work on all tests make sure they are working. Also the benches. After that, I'll do a rebase etc. And make sure your code is also working.

Reply:
It is just an WIP so don't expect you to do a full review, you can just do some quick reviews if you have time it will be opened for at least the comming week.